build/release.mk

```diff
@@ -0,0 +1,6 @@
+RELEASE_SCRIPT_PATH ?= ./tools/release.sh
+
+# Example usage: make release version=0.11.0
```

Comment:
Is there a way that we can require `version` as an argument?  This is me not knowing make well enough.

Reply:
It is required. If it's not passed in, an error message is displayed to the user.

Reply:
Right, of course.